doc/releases/changelog-dev.md

```diff
@@ -4,71 +4,73 @@
-  [(#4802)](https://github.com/PennyLaneAI/pennylane/pull/4802)
-
-* `qml.transforms.decompositions.sk_decomposition` method implements the Solovay-Kitaev algorithm for
-  approximately decomposing any single-qubit operation to Clifford+T basis.
-  [(#4801)](https://github.com/PennyLaneAI/pennylane/pull/4801)
+<h4>Statistics and drawings for mid-circuit measurements 🎨</h4>
 
-<h4>Transforms (TODO: better title) 🤖</h4>
-
-<h4>Use an iterative approach for quantum phase estimation 🔄</h4>
-
-* Iterative Quantum Phase Estimation is now available from `qml.iterative_qpe`.
-  [(#4804)](https://github.com/PennyLaneAI/pennylane/pull/4804)
+* Mid-circuit measurements can now be visualized with the text-based `qml.draw()` and the 
+  graphical `qml.draw_mpl()`.
+  [(#4775)](https://github.com/PennyLaneAI/pennylane/pull/4775)
+  [(#4803)](https://github.com/PennyLaneAI/pennylane/pull/4803)
+  [(#4832)](https://github.com/PennyLaneAI/pennylane/pull/4832)
+  [(#4901)](https://github.com/PennyLaneAI/pennylane/pull/4901)
+  [(#4850)](https://github.com/PennyLaneAI/pennylane/pull/4850)
+  [(#4917)](https://github.com/PennyLaneAI/pennylane/pull/4917)
 
-  The subroutine can be used similarly to mid-circuit measurements:
+  Mid-circuit measurement capabilities including qubit reuse and reset, postselection, and
+  conditioning are supported.
 
   ```python
-
   import pennylane as qml
 
-  dev = qml.device("default.qubit", shots = 5)
-
-  @qml.qnode(dev)
   def circuit():
+      m0 = qml.measure(0, reset=True)
+      m1 = qml.measure(1, postselect=1)
+      qml.cond(m0 - m1 == 0, qml.S)(0)
+      m2 = qml.measure(1)
+      qml.cond(m0 + m1 == 2, qml.T)(0)
+      qml.cond(m2, qml.PauliX)(1)
+  ```
+  
+  The text-based drawer outputs:
 
-    # Initial state
-    qml.PauliX(wires = [0])
-
-    # Iterative QPE
-    measurements = qml.iterative_qpe(qml.RZ(2., wires = [0]), ancilla = [1], iters = 3)
-
-    return [qml.sample(op = meas) for meas in measurements]
+  ```pycon
+  >>> print(qml.draw(circuit)())
+  0: ──┤↗│  │0⟩────────S───────T────┤  
+  1: ───║────────┤↗₁├──║──┤↗├──║──X─┤  
+        ╚═════════║════╬═══║═══╣  ║    
+                  ╚════╩═══║═══╝  ║    
+                           ╚══════╝    
   ```
+  
+  The graphical drawer outputs:
 
   ```pycon
-  >>> print(circuit())
-  [array([0, 0, 0, 0, 0]), array([1, 0, 0, 0, 0]), array([0, 1, 1, 1, 1])]
+  >>> print(qml.draw_mpl(circuit)())
   ```
+  
+  <img src="https://docs.pennylane.ai/en/latest/_images/mid-circuit-measurement.png" width=70%/>
 
-  The i-th element in the list refers to the 5 samples generated by the i-th measurement of the algorithm.
+<h4>Catalyst is seamlessly integrated with PennyLane ⚗️</h4>
 
-<h4>Drawing and statistics for mid-circuit measurements 🎨</h4>
+Catalyst, our next-generation compilation framework, is now available within PennyLane
+rather than a standalone package, allowing you to access the benefits of just-in-time (JIT)
+compilation in PennyLane proper.  
```

Comment:
```suggestion
* Catalyst, our next-generation compilation framework, is now accessible within PennyLane,
  allowing you to more easily benefit from hybrid just-in-time (JIT) compilation.
```
Minor tweak - just want to avoid the impression that Catalyst comes with PennyLane, otherwise
why do we then have to do a separate install?

Reply:
Also want to follow the "everything is a bullet point" precedent we've been following in the release notes.